website/docs/guides/apache.md

```diff
@@ -69,6 +69,7 @@ services:
       - DATABASE_PASS=${TM_DB_PASS}
       - DATABASE_NAME=${TM_DB_NAME}
       - DATABASE_HOST=database
+      - DATABASE_SSLMODE=disable
```

Comment:
Since this is the default value, I vote not to include it in the documentation at this location.